rfcs/detecting_sensitive_textual_content/20230309-implementation_plan_sensitive_terms_list.md

```diff
@@ -0,0 +1,158 @@
+
+## Outline of the approach
+
+The Openverse team will maintain a standalone public repository which contains
+an English-language list of sensitive terms. This term list is a living document
+which accepts community revisions for adding and removing new terms. The
+contents of the list will initially be based on an existing public list,
+modified at the discretion of Openverse maintainers interested in contributing.
+The list will be iterated on and refined as we develop community standards
+around what constitutes "sensitive content" for Openverse users.
+
+### Anticipated questions
+
+1.  Why is the list in a standalone repository, instead of the
+    https://github.com/wordPress/openverse?
+
+    The main reason to keep the list seperate is contributor safety. The list is
+    kept independently from the codebase so that users must explicitly consent
+    to viewing and downloading it. Additionally, there are Search Engine
+    Optimization and blocklist considerations with public GitHub repositories.
+
+    We do not, for example, want students to be unable to access our repository
+    and contribute to it because it is blocked by a school administration's web
+    filter.
+
+2.  Why is the list only in English?
+
+    At the time of writing Openverse doesn't do any translation of the textual
+    contents of indexed media. We also do not translate user-submitted searches.
+    The vast majority of indexed works are in English. Additionally, it would be
+    difficult to validate suggestions for terms in languages not spoken by
+    Openverse maintainers, which to my current knowledge includes English,
+    Spanish, Brazilian Portugese, Turkish, and Russian. English is the only
+    language used by all maintainers.
+
+3.  On what basis will Openverse maintainers use their "discretion" to create
+    this list?
+
+    Until our community has developed its own standards around sensitivity, we
+    can use a few existing documents in the WordPress and user-submitted content
+    ecosystem to guide us:
+
+    - WordPress's page on [etiquette](https://wordpress.org/about/etiquette/) in
+      the project.
+    - WordPress's
+      [code of conduct](https://make.wordpress.org/community/handbook/wordcamp-organizer/planning-details/code-of-conduct/)
+      for in-person events.
+    - YouTube's
+      [community guidelines](https://www.youtube.com/howyoutubeworks/policies/community-guidelines/)
+      offers a decent list of categories for unacceptable types of content. For
+      example, their
+      [thumbnail policy](https://support.google.com/youtube/answer/9229980?hl=en&ref_topic=9282679#zippy=%2Cage-restricted-thumbnails-and-thumbnail-removal)
+      includes some information on types of sexual and violent content that
+      either can't be used or can be used behind a flag comparable to ours.
+      - Additionally, see the [#this-will-not-be-perfect](#this-wont-be-perfect)
+        section for additional
```

Comment:
Editorial.

```suggestion
    - Additionally, see the [#this-will-not-be-perfect](#this-will-not-be-perfect)
      section for additional
```

Fixing the indentation here (maybe?) but also noting that the phrase appears incomplete.

Perhaps just deleting the "section for additional" would be fine. In context it makes sense with the name of the section referenced:

```suggestion
  - Additionally, see the [#this-will-not-be-perfect](#this-will-not-be-perfect)
```

I've also updated the heading reference in the link location as it was using `wont` instead of `will-not` and didn't match the actual heading.

Reply:
Fixed in 950bc718, thanks.